docker/docs/Dockerfile

```
@@ -12,18 +12,34 @@ RUN apt-get update \
     && Rscript \
         -e 'repos <- c(getOption("repos"),' \
         -e '           sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
+        -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
         -e 'remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
         -e 'remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
         -e 'remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
+        -e 'install.packages("pkgdown")' \
```

A little tweak to simplify future debugging: Install pkgdown from the same repositories as the packages above.
```suggestion
        -e 'install.packages("pkgdown", repos = repos)' \
```